GameData/B9_Aerospace_HX/B9_Aerospace_HX-Tank_Types.cfg

```diff
@@ -40,4 +52,35 @@ B9_TANK_TYPE
 		name = ElectricCharge
 		unitsPerVolume = 20
 	}
+}
+
+B9_TANK_TYPE
```

Comment:
Should have `:NEEDS` for when the resources don't exist

Reply:
I'll get on that.